pepper-apis/src/main/java/org/broadinstitute/ddp/elastic/ElasticSearchQueryExecutor.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.ddp.elastic;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.index.query.QueryBuilder;
+
+
+/**
+ * Executes query to ElasticSearch.
+ */
+public class ElasticSearchQueryExecutor<T> {
```

Comment:
I like the name of this class, which makes it easy to tell what exactly its role is (and to indicate that other classes do not have this particular role)

Reply:
Thanks @charlottedye! I decided to make more better design of this class (so that it could return not only map with results but also list with result rows (added another method for this purpose). And added comments with code sample how to use this class (ElasticSearchQueryExecutor).